src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/BaseProjectConfigurationDimensionProvider.cs

```diff
@@ -117,6 +137,7 @@ public IProjectXmlAccessor ProjectXmlAccessor
             {
                 var dimensionValues = ImmutableArray.CreateBuilder<KeyValuePair<string, IEnumerable<string>>>();
                 dimensionValues.Add(new KeyValuePair<string, IEnumerable<string>>(DimensionName, values));
+                TelemetryService?.PostProperty($"{DimensionName}/Get", "Value", string.Join(";", values.Select(v => HashValueIfNeeded(v))));
```

Comment:
I'd prefer a strong-typed pattern for this all location in a single location - maybe extension methods, or static methods? That way they don't get out of sync with each other.